onnxruntime/__init__.py

```diff
@@ -26,4 +28,10 @@
 except ImportError:
     pass
 
+package_name, cuda_version = '', ''
+if has_ortmodule:
+    from onnxruntime.capi.onnxruntime_validation import package_name, version, cuda_version
+    if version:
+        __version__ = version
```

Comment:
Better to check if we want to do this for inference as well.

Reply:
Would it be easier to move this check to `ortmodule/__init__.py`, so that we don't have to do this `has_ortmodule` dance?

Reply:
yeah that is right, we do not need this check of has_ortmodule. removed